.github/workflows/main.yml

```diff
@@ -1,5 +1,5 @@
 name: Checks
-on: [push]
+on: [pull_request, push]
```

Comment:
I'd recommend filtering down the branches that run for `push` otherwise it'll double run for every PR made from the repository itself (triple if you include post merge!)

Reply:
Yeah I was thinking about that myself; will do. Thanks @asottile!